clients/venice-push-job/src/main/java/com/linkedin/venice/jobs/ConfigSetter.java

```diff
@@ -0,0 +1,6 @@
+package com.linkedin.venice.jobs;
+
+@FunctionalInterface
+public interface ConfigSetter {
```

Comment:
Can we move this inside `abstract class` `clients/venice-push-job/src/main/java/com/linkedin/venice/jobs/DataWriterComputeJob.java`?